### DISCUSSION TITLE
[Feature] Tagging People

### DISCUSSION THREAD

**Discussion by BumbleCast** [immich-app/immich#11710] [Feature Request]
Since we are already able to assign birthdays to people in Immich, it would be highly beneficial to add the ability to tag people as well. This feature would enhance the functionality of the people page, making it easier to group, filter, and distinguish individuals.

For users who have lived or worked in multiple places, the people page can become quite cluttered. Currently, there is very limited control over how individuals are organized or displayed, making the people page a nice to have, but not too useful feature.

By introducing tags, users could: Group people by location (e.g., work, country, social stuff, etc), filter individuals based on tags, making it easier to find specific group, distinguish between different categories of people.

Thanks for all the great work!

**Comment by bo0tzz** [comment]
#6079
